src/main/java/edu/greenblitz/robotName/commands/arm/roller/ReleaseNote.java

```diff
@@ -0,0 +1,16 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.roller.RollerConstants;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class ReleaseNote extends SequentialCommandGroup {
+
+    public ReleaseNote(){
+        super(
+                new RunRollerUntilObjectOut(),
+                new WaitCommand(RollerConstants.TIME_UNTIL_NOTE_EXIT_ROLLER),
```

Comment:
Why do we need to wait? Roll until note is out should do that

Reply:
to make it sure